salt/returners/appoptics_return.py

```diff
@@ -187,7 +177,7 @@ def _state_metrics(ret, options, tags):
 
     log.debug(
         "Batching with  Metric total states {}".format(
-            (stats["num_failed_states"] + stats["num_passed_states"])
+            stats["num_failed_states"] + stats["num_passed_states"]
```

Comment:
It should instead read:
```python
    log.debug(
        "Batching with Metric total states %s",
        stats["num_failed_states"] + stats["num_passed_states"]
    )
```

Reply:
Is there a tool out there that can automatically update these via pre-commit?

This PR only touches this code because of the pre-commit, and the bug it fixes is really awful for anyone who needs to use the logs.  Please let's do what it takes to get this into the point release.

Reply:
Enable http://pylint.pycqa.org/en/latest/technical_reference/features.html?highlight=w1202#logging-checker-messages, rather than manual review of these?

Reply:
> Is there a tool out there that can automatically update these via pre-commit?

Unfortunately, no.

> Please let's do what it takes to get this into the point release.

It's not getting out of the point release. If you can't make the change, I'm happy to do it.

Reply:
Done, fixed them all in this file. I'd forgotten Thomas had given me push access on his fork.

What do you think about [adding `W1202`](https://github.com/saltstack/salt/pull/57723#discussion_r517343286) so this doesn't have to be addressed arbitrarily in the future?

Reply:
Unless we're explicitly disabling those messages, it's enabled by default and PyLint somehow missed them.

If they're disabled, yeah, we should enable them.

Reply:
They are disabled.

https://github.com/saltstack/salt/blob/a2f6e1bcf0a7728202f061c041e011d6bf553f7a/.pylintrc#L106